Test Suites/Individual Test Cases/License Application/Appeal/Submit appeal with invalid denial code from Public link.ts

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <TestSuiteEntity>
    <description>Submit appeal with invalid denial code from Public link</description>
-   <name>Appeal Denied License Application - Invalid denial code</name>
+   <name>Submit appeal with invalid denial code from Public link</name>
```

Comment:
@fedecl-ot please move the Test Suite to the folder "Test Suites/Individual Test Cases/Appeal/". The folder Appeal should be on the "Individual Test Case" folder, not inside of "License Application"

Reply:
Done